spec/contracts/placeholder_users/shared_contract_examples.rb

```diff
@@ -33,41 +33,50 @@
 shared_examples_for 'placeholder user contract' do
   let(:placeholder_user_name) { 'UX Designer' }
 
-  it_behaves_like 'contract is valid for active admin users only'
+  context 'when user with global permission' do
+    let(:current_user) { FactoryBot.create(:user, global_permission: %i[add_placeholder_user]) }
 
-  context 'name' do
-    context 'is valid' do
-      it_behaves_like 'contract is valid'
-    end
+    it_behaves_like 'contract is valid'
+  end
 
-    context 'is not too long' do
-      let(:placeholder_user) { PlaceholderUser.new(name: 'X' * 257) }
+  it_behaves_like 'contract is valid for active admin users only'
 
-      it_behaves_like 'contract is invalid'
-    end
+  describe 'validations' do
+    let(:current_user) { FactoryBot.build_stubbed :admin }
+    context 'name' do
+      context 'is valid' do
+        it_behaves_like 'contract is valid'
+      end
 
-    context 'is not empty' do
-      let(:placeholder_user) { PlaceholderUser.new(name: '') }
+      context 'is not too long' do
```

Comment:
I probably introduced this bug. The context here should be called 'is too long' instead of 'is not too long'.